releases/release-1.27/release-notes/maps/pr-112624-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 112624
+releasenote:
+  text: 'kubelet: fixed a bug of stoping rendering configmap when enabling fsquota
```

Comment:
"kubelet: fixed a bug in `kubelet` that stopped rendering the `ConfigMaps` when `fsquota` monitoring is enabled" might sound bette ? cc @AnaMMedina21 @yrs147 @rjsadow